modules/hitl/src/views/full/sidebar/style.scss.d.ts

```diff
@@ -1,14 +1,18 @@
 // This file is automatically generated.
 // Please do not change this file!
 interface CssExports {
-  'allPaused': string;
-  'current': string;
-  'empty': string;
-  'filter': string;
-  'header': string;
-  'sidebar': string;
-  'toggle': string;
-  'users': string;
+  allPaused: string
```

Comment:
did you manually modified the file or is it your auto-format?

Reply:
this is auto-formatting